tools/Replay.hpp

```diff
@@ -409,8 +408,10 @@ class StressSocketHandler : public WebSocketHandler
 
     static void addPollFor(SocketPoll &poll, const std::string &server,
                            const std::string &filePath, const std::string &tracePath,
-                           const std::shared_ptr<Stats> &optStats = nullptr)
+                           const std::shared_ptr<Stats> &optStats)
```

Comment:
Perhaps rename this optStats to just stats in a follow-up PR, given that it's no longer optional? Or even perhaps try to turn it into a reference if it's never nullptr.